deployment/Rules-Publish-TST-To-PRD.md

```diff
@@ -0,0 +1,48 @@
+# Rules Publish from TST to PRD
+
+1.	Remote desktop into **TDCI01SQL**
+
+2.	Using SQL Server Management Studio connect to the **TDCI01SQL** database server
+
+3.	Execute (*"Start Job at Step..."*) the SQL Server Agent Job *“Backup CodeDev”*
+
+![Backup CodeDev](../images/BackupCodeDevJob.png)
+
+4.	Open a command prompt and copy the file below into the window and hit enter
+    1.	C:\Infor\CopyDatabase - codeDEV to PRD.bat
+    ![Windows Batch File Copy Database](../images/WindowsBatchFileCopyDatabaseCodeDevToPrd.png)
```

Comment:
@Izzmo images work for me in the preview of Visual Studio Code? Where are you seeing its not working?